hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/parse/UnsupportedOperationVisitor.java

```diff
@@ -301,10 +305,6 @@ private void processCall(SqlCall call) {
     }
 
     private void processSelect(SqlSelect select) {
-        if (select.hasOrderBy()) {
```

Comment:
Do we handle unsupported parts of the `ORDER BY` (LIMIT/OFFSET, NULLS FIRST/LAST)?

Reply:
We have unit tests for the not supported parts https://github.com/hazelcast/hazelcast/pull/18013/commits/5ac898c968033856201ea1e715b5a135aaea8198
LIMIT/OFFSET will come in the next PR.